dashboard/app/models/user.rb

```diff
@@ -233,6 +233,26 @@ class User < ActiveRecord::Base
     end
   end
 
+  #
+  # TEMPORARY: Remove these aliases
+  #
+
+  def primary_authentication_option
```

Comment:
❤️ these aliases

Reply:
Note, this can be much more condensed with [alias_attribute](https://apidock.com/rails/Module/alias_attribute)

```ruby
alias_attribute :primary_authentication_option, :primary_contact_info
```